app/models/reservation_frame.rb

```diff
@@ -5,4 +5,9 @@ class ReservationFrame < ApplicationRecord
 
   validates :date, presence: true
 
+  scope :is_deleted, -> { where(:is_deleted => false) }
```

Comment:
[nits] hash は { シンボル: 値 } が一般的です。

```suggestion
  scope :is_deleted, -> { where(is_deleted: false) }
```

Reply:
is_deleted なのに削除されていないものを取得しているので、意味が逆だと思います。

Reply:
こちらで対応しました！
https://github.com/nisimotoryoga/pbl_fp_matching/pull/60/commits/acdbb4b14e089c2ba2464472d6d05fba6054fd35